src/components/Changelog/index.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { useRequest } from '../../hooks/useRequest';
+import { Heading } from '../Heading';
+import { Link } from '../Link';
+import { Spinner } from '../Spinner';
+import { Markdown } from '../../extra/Markdown';
+
+export interface ChangelogProps {
+	getChangelog: () => Promise<string>;
+}
+
+const markdownComponentOverrides: React.ComponentProps<
+	typeof Markdown
+>['componentOverrides'] = {
+	a: (props) => <Link blank {...props} />,
+	h1: (props) => <Heading.h1 {...props} fontSize="20px" />,
+	h2: (props) => <Heading.h2 {...props} fontSize="16px" />,
+	h3: (props) => <Heading.h3 {...props} fontSize="14px" />,
+	h4: (props) => <Heading.h4 {...props} fontSize="12px" />,
```

Comment:
let's try to use the theme when possible 
https://github.com/balena-io-modules/rendition/blob/ff4c9e135b7bd4936b5793be5e22eef058f65161/src/theme.ts#L119